City-of-Helsinki/helsinki-paatokset
# PP-449: Mobile fixes

**To test**
- Checkout branch, run `make drush-cim drush-cr` (Or `make new`)
- Run the following commands to pull in test data:
```
drush ap:update cases hel-2019-009216 -v
drush ap:update decisions 289da3a0-46ac-4a67-b0d9-1021e11f89a2 -v
drush ap:update decisions f43030f5-3231-490e-be04-e721c763eb91 -v
drush mim ahjo_meetings:all --update
drush ap:ud -v
```
- The favicon should now be visible on all pages
- Edit the default texts: https://helsinki-paatokset.docker.so/fi/admin/config/system/default-texts
  - Scroll down to "Frontpage calendar notice text" and input "Esityslista julkaistaan yleensä kokousta edeltävänä torstaina.", then save the settings 
- Create a landing page with the URL alias `/etusivu`: https://helsinki-paatokset.docker.so/fi/node/add/landing_page
  - Add a "News list" and a  "Meetings calendar" component to the content area. You don't need to select any filters for the news list component
- When looking at the page 
- Check the following in mobile view:
  - The news list and the meetings calendar component should have similar margins (compare to paatokset.hel.fi to see how it is currently mismatched)
- Open this decision page in mobile view: https://helsinki-paatokset.docker.so/fi/asia/hel-2022-005989/289da3a0-46ac-4a67-b0d9-1021e11f89a2
  - The main content margins should be similar to the frontpage (16px on both sides and not 32px like on https://paatokset.hel.fi/fi/asia/hel-2022-005989/289da3a0-46ac-4a67-b0d9-1021e11f89a2)
  - Open the voting results and appeal process accordion elements. The page width should stay the same instead of expanding
- Open this decision page on mobile view: https://helsinki-paatokset.docker.so/fi/asia/hel-2019-009216/f43030f5-3231-490e-be04-e721c763eb91 
  - "Suunnittelutarveratkaisuhakemus" in the title should now be split into multiple rows instead of breaking the layout
